### ISSUE TITLE
Improvements before V2 release...

### ISSUE THREAD

**Issue by THEONEANDONLYPLANE** [mSparks43/747-400#576]
Feaure Request

#1- TO/GA Throttle Lock as FMS option
In a perfect world, one would hit TO/GA on their joysticks, and their throttles would move with the in-sim movements of the throttles. Unfortunately, many throttles do not have this feature. Many engage TO/GA, accidentally touch their throttles, or try to advance the throttles after engaging TO/GA, which results in an abrupt halt to TO/GA power. Adding an option for "A/T Throttle Lock" would prevent this. Throttle lock would lock the in-sim position of the throttles when A/T is engaged, and one can only move the throttles if A/T is disengaged.

#2- Correct Panel States on Startup
Currently, when one loads the 747 into the simulator with cold and dark, engines on, or on short final, the panel states are incorrect. For example, cold and dark 747s would have their BUS TIES already pushed in, GEN CONT pushed in, etc. When loading into an engine-on state, you would expect the NAV lights on, Beacon lights on both, Recirculation fans to be on, Equipment Cooling to be on Normal, Autostart to be on, Window Heat to be on, etc. And of course, THE GEAR SHOULD BE DOWN!!! Loading into an approach state, landing lights should be on, along with other lights mentioned before, switches in various positions mentioned above, F/D's to be on, A/T to be armed, TCAS to be set to TA/RA, Lower EICAS to be cleared of any pages (I believe this is variable, but was commonly done after checklists were completed), and for a 3NM approach, gear in the DOWN position. A solution to this may be adding panel states in the FMC, or adding automatic panel states when loading into the simulator.

**Comment by crazytimtimtim** [comment]
Panel States shouldn't be difficult, just might be a lot of work as we have to set _all_ those switches... Otherwise, it should be simple.

**Comment by oMrSmith** [comment]
A/T lock is already working
but only for the thrust mode SPD.

It would be great if it could work the same way
for the thrust mode THR REF.

And: If the A/T could only be unlocked
during thrust mode HLD.

**Comment by THEONEANDONLYPLANE** [comment]
Thought I'd also add this...
Aircraft configurations such as disabling PA does not stay permenant- it resets every time, and therefore it only lasts for one flight. It would be great for those changes to stay until changes are made by the user.

**Comment by crazytimtimtim** [comment]
> Thought I'd also add this...
> Aircraft configurations such as disabling PA does not stay permenant- it resets every time, and therefore it only > lasts for one flight. It would be great for those changes to stay until changes are made by the user.

Yep, I'm going to fix that soon! I just need to figure out how @Marauder2882 simconfg system works (which I think I've mostly figured out already) or have him do it.

**Comment by Matt726-S** [comment]
surprised i didnt get another lecture about the rr engine sounds
here’s what the panel should look like (except for the battery, external power, and exterior lights)
![66373112-120D-495B-83DD-2AF2362953E1](https://user-images.githubusercontent.com/62818033/120485253-e8458600-c381-11eb-8784-4027fa07d72d.jpeg)

**Comment by crazytimtimtim** [comment]
> surprised i didnt get another lecture about the rr engine sounds

Those definitely still need improvement.

> ![66373112-120D-495B-83DD-2AF2362953E1](https://user-images.githubusercontent.com/62818033/120485253-e8458600-c381-11eb-8784-4027fa07d72d.jpeg)

Looking at that makes me think the cockpit might be a _bit_ too orange now.

**Comment by Matt726-S** [comment]
that looks like an older plane. depends on age

**Comment by Marauder2882** [comment]
> A/T lock is already working
> but only for the thrust mode SPD.
> 
> It would be great if it could work the same way
> for the thrust mode THR REF.
> 
> And: If the A/T could only be unlocked
> during thrust mode HLD as I proposed in
> [#463 (comment)](https://github.com/mSparks43/747-400/issues/463#issue-833047981)

THR REF throttle lock and HLD unlock (during TO roll) should work correctly when flying with GE engines selected in the FMC.  I put effort into including those during my design of the EEC module that was recently released but it's currently only working for GE engines.

**Comment by oMrSmith** [comment]
Hey @Marauder2882, pardon me: Could you enlighten me,
what exactly the designation of the GE engine in the FMC is?

**Comment by oMrSmith** [comment]
So I just picked a N1-type engine and the thrust setting changed according to the flight phase. 
Great stuff! Thanks to all who deliver the code!

**Comment by crazytimtimtim** [comment]
> designation of the GE engine in the FMC is?

What do you mean? Which one is GE? All the ones that start with CF6.

**Comment by oMrSmith** [comment]
That's what I meant. Thanks for the info.

**Comment by oMrSmith** [comment]
What does GE stand for anyway?

**Comment by crazytimtimtim** [comment]
General Electric.

**Comment by oMrSmith** [comment]
The selected temperature (THR REF) has no impact yet, has it?

**Comment by Marauder2882** [comment]
Correct.  Assumed Temperature has not been implemented yet for any engine type.

**Comment by crazytimtimtim** [comment]
I'll fix as many of these switch states as possible before release. Pretty much done with sound options, save a bit more testing. I'll keep the "save" and per-livery sounds for after release, so I should have time to do this.
